runtime/dotnet/azurefunctions/appsettings.json

```diff
@@ -1,9 +1,10 @@
 {
   "feature": {
+    "UseTelementryLoggerMiddleware": false,
     "UseTranscriptLoggerMiddleware": false,
     "UseShowTypingMiddleware": false,
     "UseInspectionMiddleware": false,
-    "UseCosmosDbPersistentStorage": false
+    "UseCosmosDb": false
```

Comment:
why this change?
also we are hoping to deprecate this flag

Reply:
This is the new name in web app as well, updated for parity. See: https://github.com/microsoft/BotFramework-Composer/blob/master/runtime/dotnet/azurewebapp/appsettings.json#L7